pkg/profiler/cpu/cpu.go

```diff
@@ -396,12 +430,19 @@ type (
 	// https://dave.cheney.net/2015/10/09/padding-is-hard
 	// TODO(https://github.com/parca-dev/parca-agent/issues/207)
 	stackCountKey struct {
-		PID           uint32
+		PID           int32
 		UserStackID   int32
 		KernelStackID int32
+		_             int32
+		Len           uint64
+		Addrs         [100]uint64
 	}
 )
 
+func (s *stackCountKey) WalkedWithDwarf() bool {
```

Comment:
```suggestion
func (s *stackCountKey) walkedWithDwarf() bool {
```
No need to export the method.